<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 63, 63)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_selectFile">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(27, 170, 222);
border-radius: 2px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="openfile_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:rgb(255,255,255);
color:black;
border: 1px solid;
border-color:  rgb(88, 205, 255);
border-radius: 2px;
}

QPushButton::hover{
color:blue;
}</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="selectfile_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>123</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;</string>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="carrier_groupbox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(27, 170, 222);
border-radius: 3px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="opencarrier_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color:rgb(255,255,255);
color:black;
border: 1px solid;
border-color:  rgb(88, 205, 255);
border-radius: 2px;
}

QPushButton::hover {
color: blue;
}</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="carrier_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>158</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;</string>
     </property>
     <property name="text">
      <string>Select Carrier</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="saveas_groupbox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-radius: 1px;
border-color: rgb(27, 170, 222);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="saveas_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;</string>
     </property>
     <property name="text">
      <string>Save As</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>180</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>70</y>
       <width>460</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color:white;
border:0px;}

QPushButton::hover{
color:blue;
}
</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="embed_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>830</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: white;
color:black;
}
QPushButton::hover{
color:blue;
}
</string>
    </property>
    <property name="text">
     <string>Embed</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_img">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>881</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/rabbit.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(27, 170, 222);
border-radius: 2px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meera</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(13, 107, 147);
padding:2px;

</string>
     </property>
     <property name="text">
      <string>Encryption</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="setpass_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color:white;
border:0px;}

QPushButton::hover{
color:blue;
}
</string>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>360</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/photos-icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/preferences-desktop-user-password-icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>500</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/white-folder-icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/Document-txt.png</pixmap>
    </property>
   </widget>
   <zorder>background_img</zorder>
   <zorder>groupBox_selectFile</zorder>
   <zorder>carrier_groupbox</zorder>
   <zorder>saveas_groupbox</zorder>
   <zorder>embed_button</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../PycharmProjects/lab/res.qrc"/>
 </resources>
 <connections/>
</ui>
